lib/libzpool/Makefile.am

```diff
@@ -99,6 +99,8 @@ KERNEL_C = \
 	vdev_raidz_math_sse2.c \
 	vdev_raidz_math_ssse3.c \
 	vdev_raidz_math_avx2.c \
+        vdev_raidz_math_neon.c \
```

Comment:
This (and the next line) use spaces. They should be tabs. (I checked the raw file; this isn't just the GitHub web reviewer being wrong).